internal/oci/runtime_oci.go

```diff
@@ -521,6 +577,42 @@ func (r *runtimeOCI) ExecSyncContainer(ctx context.Context, c *Container, comman
 
 	// We don't need childPipe on the parent side
 	childPipe.Close()
+	childStartPipe.Close()
+
+	if r.execCgroup && r.config.InfraCtrCPUSet != "" {
+		// Update the exec's cgroup
+		containerPid, err := c.pid()
```

Comment:
I don't think this is the pid we want. we write to a pidFile above, I think the pid we want is there

Reply:
Wouldn't pidFile contain the pid for the exec probe, which we're getting from cmd.Process.Pid for the call to moveExecToCgroup? We're getting the container pid here to lookup the pod's cgroup.

Reply:
ah I see, makes sense to me